cmd/notary/tuf.go

```diff
@@ -464,11 +466,19 @@ func readRootKey(rootKeyFile string, retriever notary.PassRetriever) (data.Priva
 	if err != nil {
 		return nil, fmt.Errorf("Error reading input root key file: %v", err)
 	}
+	isEncrypted := true
 	if err = cryptoservice.CheckRootKeyIsEncrypted(pemBytes); err != nil {
-		return nil, err
+		if role == data.CanonicalRootRole {
+			return nil, err
+		}
+		isEncrypted = false
+	}
+	var privKey data.PrivateKey
+	if isEncrypted {
+		privKey, _, err = trustmanager.GetPasswdDecryptBytes(retriever, pemBytes, "", data.CanonicalRootRole)
```

Comment:
Group review:  as a note, this whole function (and the subsequent `AddKey` calls) can probably be replaced with `utils.ImportKeys` at a later point (not necessarily in this PR - just wanted to note it here).